### PR TITLE
css/link-linearprogress

### LINKED ISSUE #1929: convert css for link

### LINKED ISSUE #1930: convert css for linearprogress

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolves #1929 
resolves #1930

### CODE REVIEW

**Review by sweeting — APPROVED** [review]
Visual check of link & linear progress look good.

File: packages/linearprogress/src/react/__specs__/index.spec.tsx [hunk 1]
```diff
@@ -20,6 +20,20 @@ describe('LinearProgress', () => {
     expect(getByTestId('mock-component')).toBeInTheDocument()
   })
 
+  it('composes className', () => {
+    const { container } = render(
+      <LinearProgress
+        data-testid="mock-component"
```
Comment (jaketrent) [inline packages/linearprogress/src/react/__specs__/index.spec.tsx:26]: nit: not needed

File: packages/linearprogress/upgradeCss.mjs [hunk 2]
```diff
@@ -0,0 +1,138 @@
+// prereq
```
Comment (jaketrent) [inline packages/linearprogress/upgradeCss.mjs:1]: remove from git

File: packages/link/upgradeCss.mjs [hunk 3]
```diff
@@ -0,0 +1,138 @@
+// prereq
```
Comment (jaketrent) [inline packages/link/upgradeCss.mjs:1]: rm from git

**Review by jaketrent — APPROVED** [review]
Overall looks good. A couple errant files to remove. Since no src code changes, I'm going to approve, but please remove those files before merge.

**Pull request merged**